File: LeetCode_log/Daily question/3239.cpp
#include<iostream>
#include<vector>

using namespace std;

int minFlips(vector<vector<int>>& grid) {
    int ansl = 0,ansr = 0,n = grid.size(),m = grid[0].size();
    for(int i = 0;i < n;++ i)
    {
        for(int j = 0;j < m / 2;++ j)
        {
            ansr += grid[i][j] != grid[i][m - 1 - j];
        }
    }
    for(int j = 0;j < m;++ j)
    {
        for(int i = 0;i < n / 2;++ i)
        {
            ansl += grid[i][j] != grid[n - 1 -i][j];
        }
    }
    return min(ansl,ansr);
}

int main()
{
    vector<vector<int>> grid = {{0,0,0},{0,0,0}};
    cout << minFlips(grid) << endl;
    return 0;
}

File: LeetCode_log/Daily question/3239.go
func minFlips(grid [][]int) int {
    ansr,ansl := 0,0
    n ,m := len(grid),len(grid[0])
    for _,med := range grid {
        for j := 0;j < m / 2;j ++ {
            if med[j] != med[m - 1 - j] {
                ansr ++
            }
        }
    }
    for j := 0;j < m;j ++ {
        for i := 0;i < n / 2;i ++ {
            if grid[i][j] != grid[n - 1 -i][j] {
                ansl ++
            }
        }
    }
    return min(ansl,ansr)
}